.github/workflows/build_native_sims.yml

```diff
@@ -0,0 +1,74 @@
+name: Build Native Sims
+
+on:
+  push:
+    branches:
+      - main
+      - reorganize_fcore
+  pull_request:
+    branches:
+      - '*'
+  schedule:
+    - cron: "0 0 * * 1"
+env:
+  test_roots: |
+    app/samples/openrocket_sensors
+    app/samples/tenants_and_tasks
```

Comment:
These are also samples, but are not in the build_samples.yaml. Can you not specify integration platforms from the sample.yaml file? For example in tenants_and_tasks I would like us to also attempt to compile for the STM32 F446, ensuring that we dont blow out our memory for example.

Reply:
yeah I would prefer to have them in samples.yml. 
there might some funny twister filters to not run anything on a specifix integration platform. j split them up originally bc by default twister doesn't know that it can't build native sim on macos/windows so it hard fails if you try